crates/client/eth-client/src/config.rs

```diff
@@ -82,6 +170,10 @@ fn default_rpc_endpoint() -> String {
     DEFAULT_RPC_ENDPOINT.into()
 }
 
+fn default_api_url() -> String {
```

Comment:
default_eth_price_api ?

Reply:
Or defaut pragma?

Reply:
default oracle api url, its not fully specific on pragma nor on eth/strk price

Reply:
But only Pragma api will return these types of values.
So if we later add support for other oracles, it will require going through some type of trait and generic.

Good enough for now tho